general/administration/hardware-acceleration.md

```diff
@@ -6,17 +6,17 @@ title: Hardware Acceleration
 # Hardware Acceleration
 
 Jellyfin supports [hardware acceleration (HWA) of video encoding/decoding using FFMpeg](https://trac.ffmpeg.org/wiki/HWAccelIntro).
-FFMpeg and Jellyfin can support multiple hardware acceleration implementations such as Intel Quicksync (QSV), AMD AMF, nVidia NVENC/NVDEC, OpenMax OMX and MediaCodec through Video Acceleration API's.
+FFMpeg and Jellyfin can support multiple hardware acceleration implementations such as Intel Quicksync (QSV), AMD AMF and nVidia NVENC/NVDEC through Video Acceleration APIs.
```

Comment:
```suggestion
FFMpeg and Jellyfin can support multiple hardware acceleration implementations such as Intel Quicksync (QSV), AMD AMF and NVIDIA NVENC/NVDEC through Video Acceleration APIs.
```
Maybe we could take this opportunity to make the NVIDIA naming more consistent. In their [branding guidelines](https://www.nvidia.com/en-us/about-nvidia/legal-info/logo-brand-usage/) they only speak of `NVIDIA` in full capitalization.